src/com/lftechnology/java/training/alina/basic/sufflenumbers/ShuffleNumbers.java

```diff
@@ -0,0 +1,88 @@
+
+package com.lftechnology.java.training.alina.basic.sufflenumbers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * <p>ShuffleNumbers Class creates an array list with 49 Integer objects from 1
+ * to 49 </p> 
+ * <p>Randomly shuffles the list </p> 
+ * <p>selects the first 6 values from the shuffled list </p>
+ * <p> sorts the selected values and prints them out
+ * </p> 
+ * <p>Use ArrayList, Collections.shuffle(), ArrayList.subList(), and
+ * Collection.sort() </p>
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class ShuffleNumbers {
+
+	private static final Logger LOGGER =
+		Logger.getLogger(ShuffleNumbers.class.getName());
+	public static List<Integer> arrayList = new ArrayList<Integer>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make arrayList a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this "public static arrayList" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1444)